auslib/dockerflow.py

```diff
@@ -26,8 +28,12 @@ def heartbeat():
         """Per the Dockerflow spec:
         Respond to /__heartbeat__ with a HTTP 200 or 5xx on error. This should
         depend on services like the database to also ensure they are healthy."""
-        database_entry_value = heartbeat_database_fn(dbo)
-        return Response(str(database_entry_value), headers={"Cache-Control": "no-cache"})
+        try:
+            database_entry_value = heartbeat_database_fn(dbo)
+            return Response(str(database_entry_value), headers={"Cache-Control": "no-cache"})
+        except Exception as e:
+            logging.exception("Error occured: %s", e.message)
+            return Response(status=502, response=e.message)
```

Comment:
In https://bugzilla.mozilla.org/show_bug.cgi?id=1287536, @mostlygeek requested that we return a 502 with a useful error message. I'm good with either, though.